AudioKit/Common/Nodes/Effects/Dynamics/Compressor/AKCompressor.swift

```diff
@@ -114,22 +115,17 @@ open class AKCompressor: AKNode, AKToggleable, AUEffect, AKInput {
         self.releaseTime = releaseTime
         self.masterGain = masterGain
 
-        input?.connect(to: inputGain)
         inputGain.volume = 0
-        mixer = AKMixer(inputGain)
-
-        effectGain = AKMixer(input)
-        effectGain?.volume = 1
+        effectGain.volume = 1
 
         let effect = _Self.effect
         self.internalEffect = effect
-
         AudioKit.engine.attach(effect)
         au = AUWrapper(effect)
-        if let node = effectGain?.avAudioNode {
-            AudioKit.engine.connect(node, to: effect)
-        }
-        AudioKit.engine.connect(effect, to: mixer.avAudioNode)
+
+        input?.connect(to: inputMixer)
```

Comment:
Fixed by adding inputMixer and splitting, and returning inputMixer for inputNode.